app/codes/updater.py

```diff
@@ -190,3 +197,89 @@ def pay_fee_for_transaction(cur, transaction):
             fee / len(payees)
         )
     return True
+
+
+def mine_empty_block():
+    con = sqlite3.connect(NEWRL_DB)
+    cur = con.cursor()
+
+    blockchain = Blockchain()
+
+    block = blockchain.mine_empty_block(cur, {'transactions': []})
+    # update_db_states(cur, block)
+
+    con.commit()
```

Comment:
just check if we are also committing changes for a normal block in a similar manner.